test/redux_test.dart

```diff
@@ -0,0 +1,9 @@
+import 'middleware_test.dart' as middleware_test;
+import 'store_test.dart' as store_test;
+import 'utils_test.dart' as utils_test;
```

Comment:
Another option for code coverage is to use [package:coverage](https://github.com/dart-lang/coverage)

Reply:
Hrm -- this PR makes use of `package:coverage` to collect the coverage, but I created this file since I wanted collect coverage from all tests in one shot.

Does the coverage package have a way to do the same thing as "pub run test"?